src/sass/blocks/header.scss

```diff
@@ -0,0 +1,227 @@
+.page-header {
+  background-color: $white;
+  height: 56px;
+
+  &__logo {
+    position: absolute;
+    margin-left: 40%;
+    width: 102px;
+    margin: auto;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    z-index: 1;
+
+  }
+}
+.nav {
+  display: flex;
+  padding: 17px 20px;
+  justify-content: end;
+
+
+
+  &__wrapper{
+    display: none;
+  }
+
+  &__toggle {
+    width: 23px;
+    height: 4px;
+    background-color: $main-blue;
+    position: relative;
+    cursor: pointer;
+
+    &::after {
+      content: "";
+      position: absolute;
+      width: 23px;
+      height: 4px;
+      background-color: $main-blue;
+      top: 8px;
+      left: 0px
+    }
+
+    &::before {
+      content: "";
+      position: absolute;
+      width: 23px;
+      height: 4px;
+      background-color: $main-blue;
+      top: 16px;
+      left: 0px
+      }
+      &:hover{
+        opacity: 0.6;
+      }
+      &:visited{
+        opacity: 0.3;
+      }
+    }
+}
+
+.nav--open {
```

Comment:
In the `open` state, the menu should appear above instead of overlapping the page.

![image](https://github.com/Alexmasya97/Sedona-adaptive/assets/77216072/ced4ab4a-ade4-44d2-b47d-c2bc9e9ff22b)